src/bit-systems/mixer-animatable.ts

```diff
@@ -10,6 +10,10 @@ const mixerExitQuery = exitQuery(mixerQuery);
 
 export function mixerAnimatableSystem(world: HubsWorld): void {
   initializeEnterQuery(world).forEach(eid => {
+    if (entityExists(world, eid)) {
```

Comment:
This line breaks mixer animation. I think here should be `!entityExists(world, eid)`